src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.Spark;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+
+public class Intake extends SubsystemBase {
+
+  private final Spark m_IntakeMotor = new Spark(Constants.IntakeMotorPort);
+
+  private final DoubleSolenoid m_IntakeSolenoid = new DoubleSolenoid(Constants.IntakeSolenoidPort1,
+      Constants.IntakeSolenoidPort2);
```

Comment:
Not something to worry about, but the way that I've been using the m_notation is only when I'm setting a variable equal to a class that was created by our team. In other words, the only things with m_ are the files that we see under java/src (m_Intake, m_RobotContainer, m_extendIntake). I don't know if that makes more sense or not.